### PR TITLE
Release ReactOS 0.4.13

### PR DESCRIPTION
**Pull request by ColinFinck**
Press release imported from https://docs.google.com/document/d/1hFrlQ9Hbmf392YtcxpLi_AZr8AOJ6euN1EGRwp4oVGQ/edit

Please have a last look before it goes live.

### CODE REVIEW

File: content/project-news/reactos-0413-released.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+In the area of accessibility in ReactOS, new software was introduced - the Accessibility Utility Manager.
+Developed by ReactOS contributor Bișoc George, the Accessibility Utility Manager, as the name implies, manages the accessibility applications of ReactOS.
+The following screenshot shows the program in operation:
+
+{{< figure link="/img/project-news/reactos-0413-released/accessibility.png" src="/img/project-news/reactos-0413-released/accessibility.png" >}}
+
+Similarly, the On-Screen Keyboard has seen some improvements to theming support and there have been some additional features and minor bug fixes in the Magnifier.
+
+# Font Management
+Contributor Katayama Hirofumi has continued his work to bring ReactOS’ font support closer to that of Windows.
```
Comment (learn-more) [inline content/project-news/reactos-0413-released.md:63]:
```suggestion
Developer Katayama Hirofumi has continued his work to bring ReactOS’ font support closer to that of Windows.
```

**Comment by JoachimHenze** [comment]
In case you are not live yet:
"significantly improving the boot speed when using this filesystem" ->
"significantly improving the boot speed when using this filesystem for USB-RAM boot.
 for USB-RAM boot." No other complaints.

File: content/project-news/reactos-0413-released.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+# USB Improvements
+USB (Universal Serial Bus) is an important standard adopted throughout the industry and used widely on many devices such as external storage, keyboards, mice, and various other devices.
+Notably, there are a great number of computers that do not come with a CD-ROM drive these days, with the only option being USB in order to boot or install an operating system.
+The USB stack itself is arguably a very complicated one, the code to support it is complex, and maintaining it is not an easy task.
+ReactOS currently lags behind an advanced USB implementation as various bugs and regressions still plague the system.
+Despite this, many people have worked hard and contributed code to improve USB functionality in ReactOS.
+
+Vadim Galyant and Victor Perevertkin have brought in a new USB stack which provides many fixes as well as improvements to various areas, improving USB support in general.
+Although there is a long journey ahead towards an advanced and perfectly functional USB stack within ReactOS, the new module is a further step towards usable and working USB.
```
Comment (Extravert-ir) [inline content/project-news/reactos-0413-released.md:23]:
```suggestion
Although there is a long journey ahead towards an advanced and perfectly functional USB stack within ReactOS, the new module already supports HID and storage devices.
```

**Comment by Extravert-ir** [comment]
Oh, headers should be `<h2>`, not `<h1>`

**Pull request merged**